docs/en/config/gnss_denied_environments.md

```diff
@@ -0,0 +1,101 @@
+# GNSS-Denied Environments
+
+PX4 can be used without a GNSS or in GNSS-denied environments, such as when signals are blocked or jammed.
+This section describes the various configuration options, and when they are needed.
+
+## Overview
+
+<!--
+
+Here is want to cover that flight modes provide features for stabilization of altitude, position, velocity and so on.
+Some of them require global position while others, require only altitude, or a local position.
+Modes that don't require global position can be used without a GNSS or in a GNSS denied environment, but you may still need to configure some default, which typically assume the presence of a GNSS.
+
+However if you don't have a GNSS or you need to be able to fly in GNSS denied environment you can still use the automatic modes in some circumstances, given another source of position.
+
+
+-->
+
+## Modes that don't require GNSS
+
+- **Manual mode** (no stabilization assistance)
+- **Stabilized mode** (attitude stabilization only)
+- **Altitude mode** (if you have a reliable barometer/rangefinder)
+
+---
+
+<!-- Below is working information for data mining.
+I want to structure so that people know when they need to make these changes, and direct them towards the stuff they need.
+-->
+
+## Primary Solution: Configure for GPS-Denied Operations
+
+**1. Enable arming without GPS:**
```

Comment:
Note, you have to do this to arm in a mode that normally requires it. If you won't fly auto you might not do this.

I am waiting on linked PR to find out why you would not set `2` if you have a VIO or other source of position, or `0` if you don't.